### PR TITLE
Trello Connector 1.0 Docs

### PR DESCRIPTION
**Pull request by dhirsch1**

### CODE REVIEW

File: trello/1.0/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* How to create a Mule app using Anypoint Studio
+
+Before creating an app, you must have access to:
+
+* The Trello target resource
+* Anypoint Platform
+* Anypoint Studio version 4.2.2 or later
```
Comment (kevintroller) [inline trello/1.0/modules/ROOT/pages/index.adoc]: 4.2.2 is the Mule runtime engine version not the version of Anypoint Studio. Could devs confirm if this connector can be used with Studio 7.8?

File: trello/1.0/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+== Common Use Cases For the Connector
+
+Use Trello Connector to manage Trello boards, lists, and cards.
+
+== Connection Types
+
+Trello connections use token-based authentication. Each Trello user is given an API key, which enables the user to generate a token. A user then passes his or her API key and token in the URL for each request.
```
Comment (kevintroller) [inline trello/1.0/modules/ROOT/pages/index.adoc]:
```suggestion
Trello connections use token-based authentication. Each Trello user is given an API key, which enables the user to generate a token. Users then passes their API key and token in the URL for each request.
```

**Review by FilipBogdanSV — DISMISSED** [review]

**Review by kevintroller — DISMISSED** [review]

**Review by valkyrie69 — APPROVED** [review]

File: trello/1.0/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Before creating an app, you must have access to:
+
+* The Trello target resource
+* Anypoint
+* Anypoint Studio version 7.3.5 or later
+
+== Common Use Cases For the Connector
+
+Use Trello Connector to manage Trello boards, lists, and cards.
```
Comment (varunmaker) [inline trello/1.0/modules/ROOT/pages/index.adoc:27]: Customer Case Management
* Send customer case information managed by your support teams to your sales, development and other teams for better and faster collaboration .

Project Collaboration
* Manage key tasks for your project teams across all your IT apps (email, messaging, calendar) in one, streamlined and easy-to-use interface. Make working across teams smoother, especially in a remote work environment.

Comment (varunmaker) [reply trello/1.0/modules/ROOT/pages/index.adoc:27]: @dhirsch1 added use cases.

**Review by varunmaker — COMMENTED** [review]
@dhirsch1 added two use cases. Could you please incorporate them? Other than that, approved.

**Pull request merged**